README.rst

```diff
@@ -132,6 +132,9 @@ Finally, execute::
 
    python setup.py install
 
+**NOTE:** 
+Only conda Python is supported at the moment, and *not* the official python.org distribution. There is likely a binary compatibility issue between the C compiler used to build IPOPT and that used for compiling Python extensions in the official Python distribution (see https://github.com/matthias-k/cyipopt/issues/52).
```

Comment:
Can you change "supported" to "tested"? We will attempt to support the official Python, but just don't have it tested in this repo.